examples/structured_2d_dgsem/elixir_euler_rayleigh_taylor_instability.jl

```diff
@@ -0,0 +1,73 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+gamma = 1.4
+equations = CompressibleEulerEquations2D(gamma)
+
+initial_condition = initial_condition_rti
+source_terms = source_terms_rti
+
+surface_flux = flux_hll
+volume_flux  = flux_ranocha
+polydeg = 3
+basis = LobattoLegendreBasis(polydeg)
+volume_integral = VolumeIntegralFluxDifferencing(volume_flux)
+solver = DGSEM(basis, surface_flux, volume_integral)
```

Comment:
```suggestion
volume_flux  = flux_ranocha
solver = DGSEM(polydeg=3, surface_flux=flux_hll,
               volume_integral=VolumeIntegralFluxDifferencing(volume_flux))
```
Shorter and IMHO easier to read (using our convenience constructors for often-changed config values)